code-challenges.js

```diff
@@ -57,12 +127,37 @@ const givenValue1 = 42
 const givenValue2 = 10
 // Expected output: 8
 
-// --------------------5) Create the code that will sort all the numbers in an array from largest to smallest. Use the test variables provided below.
+// console.log(myNumbers.length - 2)
+// console.log(myNumbers.length - 1)
+
+// console.log(myNumbers.lastIndexOf(givenValue1))
+// console.log(myNumbers.lastIndexOf(givenValue2))
+
+
+// The first code works, but I feel like I am not understanding how to use the variables "givenValue1, givenValue2" to run my code without subtracting from the total index of myNumbers. 
+// I went back and researched other options and found the .lastIndexOf method (from Javascripttutorials.net), and I was able to use all the given Data in my code.
+
+
+
+
+//  --------------------5) Create the code that will sort all the numbers in an array from largest to smallest. Use the test variables provided below.
 
 // Pseudo code:
+// input: all numbers in the given arrays
+// output: rearranged arrays with all index items sorted from largest to smallest
+
+// process: 
+// sort the given arrays in numerical order
+// then make them largest to smallest
+
 
 const sanDiegoSummerTemperatures = [79, 80, 72, 73, 82, 77, 76]
 // Expected output: [82, 80, 79, 77, 76, 73, 72]
 
 const sanDiegoWinterTemperatures = [59, 68, 62, 59, 66, 67, 66]
 // Expected output: [68, 67, 66, 66, 62, 59, 59]
+
+// console.log(sanDiegoSummerTemperatures.sort().reverse())
+// console.log(sanDiegoWinterTemperatures.sort().reverse())
+
+// code worked, is there a way to make the .sort command go in decending order without .reverse?
```

Comment:
Great question! There is a way we can use a function in the sort method to change the way it sorts. Try to look into it and feel free to reach out if you don't understand!